### ISSUE TITLE
AnymailRequestsAPIError: 

### ISSUE THREAD

**Issue by pdpinch** [mitodl/open-discussions#1524]
I'm not really sure how this happened, but we have an email address with a bad domain name in discussions. It came from MicroMasters, so edX has to be the original source. 

Accounts on edX (and discussions) are normally validated, so I don't know exactly how this happened. Perhaps the edX user changed their email from a valid one to an invalid one. 

It's pretty obvious in this case what the correct email should be, but I'm not sure that repairing the data is the right thing to do. 

https://sentry.io/mit-office-of-digital-learning/discussions/issues/751465001/?referrer=github_plugin

```
AnymailRequestsAPIError: Sending a message to Binghui Chen <chenbinghui105@foxmail.copm> from MIT Open Learning <no-reply@discussions-mail.odl.mit.edu>
Mailgun API response 400: BAD REQUEST
{
  "message": "'to' parameter is not a valid address. please check documentation"
}
(5 additional frame(s) were not displayed)
...
  File "anymail/backends/base.py", line 93, in send_messages
    sent = self._send(message)
  File "anymail/backends/base_requests.py", line 55, in _send
    return super(AnymailRequestsBackend, self)._send(message)
  File "anymail/backends/base.py", line 123, in _send
    response = self.post_to_esp(payload, message)
  File "anymail/backends/base_requests.py", line 77, in post_to_esp
    self.raise_for_status(response, payload, message)
  File "anymail/backends/base_requests.py", line 89, in raise_for_status
    backend=self)

Error sending email '[Open Learning] Top Stories for You' to REDACTED
```

**Comment by rhysyngsun** [comment]
This account probably came through before MM validated the account validity itself. Technically neither the MM nor discussions accounts should've ever been created.

I don't think we'd want to disable the user or anything, maybe the best solution would be to have a `MailgunRefused` table where if a record exists for a user, we don't send emails to them, and we add the `message` to that table as well so we can at least see these in django-admin (and they can be managed there).

**Comment by pdpinch** [comment]
closed via #2148
